test/lit.site.cfg.in

```diff
@@ -76,6 +74,12 @@ if "@CMAKE_GENERATOR@" == "Xcode":
     config.environment['PATH'] = \
       os.path.pathsep.join((xcode_bin_dir, config.environment['PATH']))
 
+if "@LLDB_ENABLE@" == "TRUE":
+    config.available_features.add('lldb')
+    for root, dirs, files in os.walk("@LLDB_BUILD_DIR@"):
+        if root.endswith("site-packages"):
+            config.substitutions.append(('%lldb-python-path', root))
```

Comment:
Also, I'm not sure I understand. Where is the `%lldb-python-path` substitution being used? Is it used anywhere? Did you perhaps forget to include the tests in your commit?